bucket/jadx.json

```
@@ -8,14 +8,17 @@
     },
     "url": "https://github.com/skylot/jadx/releases/download/v1.5.2/jadx-1.5.2.zip",
     "hash": "5a8b480839c9c61527895d81d5572182279d973abe112047417f237df958a3aa",
+    "pre_install": "Invoke-WebRequest -Uri https://raw.githubusercontent.com/skylot/jadx/refs/heads/master/jadx-gui/src/main/resources/logos/jadx-logo.ico -OutFile $dir\\jadx-logo.ico",
```

```suggestion
    "pre_install": [
        "# Extract app icon from bundled JAR",
        "Expand-7zipArchive -Path \"$dir\\lib\\jadx-$version-all.jar\" -DestinationPath \"$dir\" -ExtractDir 'logos'"
    ],
```

Extract icon from JAR instead download from internet.

---

done